src/WebJobs.Extensions.NotificationHubs/NotificationHubAttribute.cs

```diff
@@ -47,5 +47,11 @@ public sealed class NotificationHubAttribute : Attribute
         /// <see cref="NotificationHubsConfiguration"/>.
         /// </summary>
         public string HubName { get; set; }
+
+        /// <summary>
+        /// Optional. Boolean value to enable debug send on NotificationHubClient
+        /// <see cref="NotificationHubClient"/>.
+        /// </summary>
+        public bool EnableTestSend { get; set; }
```

Comment:
I was thinking about this like `RunOnStartup` for timers -- which I always think of as a debugging switch. You're likely going to want to debug a single function with this setting rather than just run debugging on across-the-board. I think it makes sense to be attribute-only.